### PR TITLE
Fetching Description from metadata

### PR DESCRIPTION
**Pull request by juliantoledo**
Fetching Description from metadata when not present in manifest
For #70

### CODE REVIEW

**Comment by juliantoledo** [comment]
I get this error when I try to run it:
`App listening on port 8080
/Users/jtoledo/GitHub/gulliver/node_modules/express/lib/router/index.js:618
    return fn.apply(this, arguments);
             ^

TypeError: Cannot read property 'apply' of undefined
    at /Users/jtoledo/GitHub/gulliver/node_modules/express/lib/router/index.js:618:14
    at Immediate.next (/Users/jtoledo/GitHub/gulliver/node_modules/express/lib/router/index.js:256:14)
    at Immediate.<anonymous> (/Users/jtoledo/GitHub/gulliver/node_modules/express/lib/router/index.js:618:15)
    at Immediate.immediate._onImmediate (timers.js:585:18)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)`

File: lib/pwa.js [hunk 1]
```diff
@@ -108,6 +122,20 @@ exports.save = function(pwa, callback) {
   });
 };
 
+exports.fetchMetadataDescription = function(url) {
```
Comment (andreban) [inline lib/pwa.js]: I'm wondering if this method and it's support libraries should be moved to a different javascript file.

**Comment by andreban** [comment]
Small note on putting the code that fetches the page and extracts the description to a different file.

File: lib/pwa.js (hunk 1 above)
Comment (juliantoledo) [inline lib/pwa.js]: I think so, I will move it.

**Pull request merged**